### ISSUE TITLE
[Bug] Unable to render dicom

### ISSUE THREAD

**Issue by syed-qure** [OHIF/Viewers#4194]
### Describe the Bug

Dicom is rendered improperly

### Steps to Reproduce

dicom [1.37.934.5070.22764.207135.1081378.75592409959270857737967299468.dcm.zip](https://github.com/OHIF/Viewers/files/15503569/1.37.934.5070.22764.207135.1081378.75592409959270857737967299468.dcm.zip)

### The current behavior

Dicom is not rendered properly

### The expected behavior

Dicom should have been rendered.

### OS

macOS 13.2.1 (22D68)

### Node version

18.19.1

### Browser

Arc v1.40.1 (49250)

**Comment by sedghi** [comment]
Yeah it does not work, try uncompressing it and try compressing it with dcm4chee toolkit or something that is standard and lets see what happens
